src/atomicgen.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python
+import re
+
+# print the header
+def print_header_file(filename):
+    header_file = ""
+    header_file += "/// \/file atomic_nuclear_data.h\n"
+    header_file += "/// \/author Andrew Davis (andrew.davis@wisc.edu)\n"
+    header_file += "///\n"
+    header_file += "/// \/brief Impliments all the fundamental atomic & nuclear data data\n"
+    header_file += "#include <map>\n"
+    header_file += "\n"
+    header_file += "namespace pyne\n"
+    header_file += "{\n"
+    header_file += "  /// main function to be called when you whish to load the nuclide data \n"
+    header_file += "  /// into memory \n"
+    header_file += "  void _load_atomic_mass_map_memory();\n"
+    header_file += "  /// function to create mapping from nuclides in id form\n" 
+    header_file += "  /// to their atomic masses\n"
+    header_file += "  \n"
+    header_file += "  void _insert_atomic_mass_map();\n"
+    header_file += "  \n"
+    header_file += "  /// function to create mapping from nuclides in id form \n"
+    header_file += "  /// to their natural abundances\n"
+    header_file += "  void _insert_abund_map();\n"
+    header_file += "  \n"
+    header_file += "  /// Mapping from nuclides in id form to their natural abundances\n"
+    header_file += "  extern std::map<int,double> natural_abund_map;\n"
+    header_file += "  \n"
+    header_file += "  /// Mapping from nuclides in id form to their atomic masses.\n"
+    header_file += "  extern std::map<int,double> atomic_mass_map;\n"
+    header_file += "  \n"
+    header_file += "  /// Mapping from nuclides in id form to the associated error in \n"
+    header_file += "  /// abdundance \n"
+    header_file += "  extern std::map<int,double> atomic_mass_error_map;\n"
+    header_file += "} // namespace pyne\n"
+    
+    f = open(filename,'w')
+    f.write(header_file)
+    f.close()
+
+# print the masses map
+def print_atomic_mass_errors():
+    atomic_mass_error = ""
+
+    file = open('../pyne/dbgen/mass.mas12','r')
+    amdc_regex = re.compile('[ \d-]*? (\d{1,3})[ ]{1,4}(\d{1,3}) [A-Z][a-z]? .*? (\\d{1,3}) ([ #.\d]{5,12}) ([ #.\d]+)[ ]*?$')
+    for line in file:
+        m = amdc_regex.search(line)
+        if m is None:
+            continue
+            
+        nuc = (10000000 * int(m.group(1))) + (10000 * int(m.group(2)))
+        error = 1E-6 * float(m.group(5).strip().replace('#', ''))
```

Comment:
Don't know, same place as in https://github.com/pyne/pyne/blob/develop/pyne/dbgen/atomic_mass.py